### PR TITLE
feat(typed_array_elements): add typed array elements snippet

### PR DESCRIPTION
**Pull request by domuserk**
This background script may be useful if you come across a situation where it would be necessary to convert elements to a specific type such as string or number.

### CODE REVIEW

File: Background Scripts/Typed Array Elements/readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Attach Workflow to existing records
```
Comment (Lacah) [inline Background Scripts/Typed Array Elements/readme.md:1]: What is this line here?

**Review by Lacah — CHANGES_REQUESTED** [review]
The readme file is a bit confusing, especially with the first line. Could you please update it so that it is obvious to less experienced users as well (who may not know how array.map works) what your script is about and how to use it?
Thanks.

File: Background Scripts/Typed Array Elements/readme.md (hunk 1 above)
Comment (domuserk) [reply Background Scripts/Typed Array Elements/readme.md:1]: its a mistake sorry

**Review by Lacah — APPROVED** [review]
Awesome, thanks!

**Pull request merged**